api/registry/models.py

```diff
@@ -204,14 +204,22 @@ class Chain(models.TextChoices):
         choices=Chain.choices, default=Chain.ETHEREUM, db_index=True
     )
     unlock_time = models.DateTimeField(null=False, blank=False)
+
+    # u256
     last_updated_in_block = models.DecimalField(
         decimal_places=0, null=False, blank=False, max_digits=78, db_index=True
     )
 
+    # u64
+    lock_duration = models.DecimalField(
```

Comment:
I would use `DurationField` here https://docs.djangoproject.com/en/5.0/ref/models/fields/#durationfield
I think this makes more sense from a DB / analytics perspective